src/qhdopt/qhd.py

```diff
@@ -171,9 +169,15 @@ def f_eval(self, x):
         x = self.jax_affine_transformation(x.astype(jnp.float32))
         return self.lambda_numpy(*x)
 
-    def classically_optimize(self, samples=None, solver="TNC"):
-        if samples is None:
-            samples = np.random.rand(self.shots, self.dimension)
+    def classically_optimize(self, shots=100, solver="TNC", verbose=0):
+        self.baseline_setup(shots, solver)
```

Comment:
In baseline_setup, the samples are initialized randomly on the discretized space (i.e., the mesh grid point). But I assume classically_optimize(...) should also be able to handle user-specified initial guesses. Or do we move this feature to some other functions?